easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -105,6 +105,109 @@ def new_ec_method(self, key, *args, **kwargs):
     return new_ec_method
 
 
+def module_is_available(full_mod_name, modtool, avail_modules, hidden):
```

Comment:
well, you can replace it with the same single line there too?

having a function like this here is kind of weird

if you want to keep it, it should be moved to `easybuild.tools.modules` instead